pkg/controller/servicemesh/controlplane/reconciler.go

```diff
@@ -465,8 +481,10 @@ func (r *ControlPlaneReconciler) postReconciliationStatus(reconciliationMessage
 	}
 	reconciledCondition := r.Status.GetCondition(v1.ConditionTypeReconciled)
 	if processingErr == nil {
+		reconciledCondition.Reason = reconciliationReason
```

Comment:
Since this is true regardless of which condition we hit, should it be external to the if?

Reply:
Of course. Thanks! Fixed.